### ISSUE TITLE
Sigmoid formula

### ISSUE THREAD

**Issue by peppemas** [dani2442/AI_Car#2]
https://github.com/dani2442/AI_Car/blob/959f86cdb9cb93d3c9a60eca3df2dddcbb884ef3/AI_Car/Source/AI_Car/NeuralNetwork.h#L34

Hi, 

why you use -1 + 2 / ( 1 + exp(-x) ) instead of 1 / (1 + exp(-x)) ?

**Comment by simon-cpp** [comment]
Yes, it should definitely be _return 1.f / (1 + FGenericPlatformMath::Exp(-x)_
Especially since it runs continuously for every node.
